2019/07/17 08:42:30 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php?r=site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:42:47 [error] [php] include(User.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php?r=site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:43:28 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:43:35 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:45:27 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:47:01 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:47:04 [error] [php] include(Users.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:463)
Stack trace:
#0 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(22): spl_autoload_call()
#1 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/site/login
in C:\xampp\htdocs\brother\protected\components\UserIdentity.php (22)
in C:\xampp\htdocs\brother\protected\models\LoginForm.php (54)
in C:\xampp\htdocs\brother\protected\controllers\SiteController.php (94)
2019/07/17 08:47:56 [error] [exception.CException] CException: Property "Users.id" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(31): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/brother/index.php/site/login
HTTP_REFERER=http://localhost/brother/index.php/site/login
---
2019/07/17 08:48:40 [error] [exception.CException] CException: Property "Users.id" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(31): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/brother/index.php/site/login
HTTP_REFERER=http://localhost/brother/index.php/site/login
---
2019/07/17 08:49:58 [error] [exception.CException] CException: Property "Users.id" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(31): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/brother/index.php/site/login
HTTP_REFERER=http://localhost/brother/index.php/site/login
---
2019/07/17 08:50:13 [error] [exception.CException] CException: Property "Users.id" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('id')
#1 C:\xampp\htdocs\brother\protected\components\UserIdentity.php(31): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\brother\protected\models\LoginForm.php(54): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), Array)
#6 C:\xampp\htdocs\brother\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/brother/index.php/site/login
HTTP_REFERER=http://localhost/brother/index.php/site/login
---
2019/07/17 08:59:49 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/view
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:10 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/view
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:13 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/view
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:13 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/view
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:13 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/view
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:16 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/update
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:28 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/update
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:29 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/update
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:00:29 [error] [exception.CHttpException.400] CHttpException: Your request is invalid. in C:\xampp\htdocs\yii\framework\web\CController.php:337
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/brother/index.php/topic1/update
HTTP_REFERER=http://localhost/brother/index.php/topic1/index
---
2019/07/17 09:13:07 [error] [exception.CException] CException: Property "CGridView.table" is not defined. in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('table', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(Topic1Controller), 'CGridView', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(181): CBaseController->createWidget('zii.widgets.gri...', Array)
#3 C:\xampp\htdocs\brother\protected\views\topic1\admin.php(22): CBaseController->widget('zii.widgets.gri...', Array)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('admin', Array, true)
#8 C:\xampp\htdocs\brother\protected\controllers\Topic1Controller.php(142): CController->render('admin', Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): Topic1Controller->actionAdmin()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/admin')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#21 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/brother/index.php/topic1/admin
HTTP_REFERER=http://localhost/brother/index.php/site/page?view=about
---
2019/07/17 09:17:31 [error] [exception.Error] Error: Call to undefined function background-color:#ffffff() in C:\xampp\htdocs\brother\protected\views\topic1\admin.php:25
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('admin', Array, true)
#4 C:\xampp\htdocs\brother\protected\controllers\Topic1Controller.php(142): CController->render('admin', Array)
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): Topic1Controller->actionAdmin()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/admin')
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/brother/index.php/topic1/admin
HTTP_REFERER=http://localhost/brother/index.php/site/page?view=about
---
2019/07/17 09:17:44 [error] [exception.Error] Error: Call to undefined function background-color:red() in C:\xampp\htdocs\brother\protected\views\topic1\admin.php:25
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#2 C:\xampp\htdocs\yii\framework\web\CController.php(872): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(785): CController->renderPartial('admin', Array, true)
#4 C:\xampp\htdocs\brother\protected\controllers\Topic1Controller.php(142): CController->render('admin', Array)
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): Topic1Controller->actionAdmin()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): CController->runAction(Object(CInlineAction))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter(Object(CFilterChain))
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('topic1/admin')
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#17 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/brother/index.php/topic1/admin
HTTP_REFERER=http://localhost/brother/index.php/site/page?view=about
---
2019/07/17 09:29:52 [error] [exception.CHttpException.404] CHttpException: Unable to resolve the request "listmenu/create". in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('listmenu/create')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\brother\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/brother/index.php/listmenu/create
HTTP_REFERER=http://localhost/brother/index.php/document/admin
---
2019/07/17 09:34:21 [error] [php] Undefined variable: form (C:\xampp\htdocs\brother\protected\views\document\_form.php:11)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(872): DocumentController->renderFile()
#1 C:\xampp\htdocs\brother\protected\views\document\create.php(18): DocumentController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): DocumentController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(872): DocumentController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(785): DocumentController->renderPartial()
#6 C:\xampp\htdocs\brother\protected\controllers\DocumentController.php(78): DocumentController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DocumentController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(134): DocumentController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1148): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): DocumentController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(131): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): DocumentController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DocumentController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(185): CWebApplication->processRequest()
#19 C:\xampp\htdocs\brother\index.php(13): CWebApplication->run()
REQUEST_URI=/brother/index.php/document/create
in C:\xampp\htdocs\brother\protected\views\document\_form.php (11)
in C:\xampp\htdocs\brother\protected\views\document\create.php (18)
in C:\xampp\htdocs\brother\protected\controllers\DocumentController.php (78)
